docs/intro/guides/drizzle/nextjs.rst

```diff
@@ -0,0 +1,271 @@
+          author: text(),
+          year: smallint(),
+          genre: text(),
+          read_date: timestamp(),
+        }, (table) => [
+          uniqueIndex("books_pkey").using("btree", table.id.asc().nullsLast().op("uuid_ops")),
+        ]);
+
+        export const notes = gelTable("Note", {
+          id: uuid().default(sql`uuid_generate_v4()`).primaryKey().notNull(),
+          text: text().notNull(),
+          created_at: timestamp().default(sql`datetime_current()`),
+          book_id: uuid().notNull(),
+        }, (table) => [
+          uniqueIndex("notes_pkey").using("btree", table.id.asc().nullsLast().op("uuid_ops")),
+        ]);
+
+
+Keep Drizzle in sync with Gel
+=============================
+
+.. edb:split-section::
+
+    To keep your Drizzle schema in sync with your Gel schema, add a hook to your ``gel.toml`` file. This hook will automatically run ``drizzle-kit pull`` after each migration:
+
+    .. code-block:: toml
+        :caption: gel.toml
+
+        [hooks]
+        after_migration_apply = [
+          "npx drizzle-kit pull"
+        ]
+
+
+
+With this hook in place, your Drizzle schema will automatically update whenever you apply Gel migrations.
+
+
+Create a database client
+========================
+
+.. edb:split-section::
+
+    Now, let's create a database client that you can use throughout your application:
+
+    .. code-block:: typescript
+        :caption: src/db/index.ts
+
+        import { drizzle } from 'drizzle-orm/gel';
+        import { createClient } from 'gel-js';
+        import * as schema from '@/drizzle/schema';
+        import * as relations from '@/drizzle/relations';
+
+        // Import our schema
+        import * as schema from './schema';
+
+        // Initialize Gel client
+        const gelClient = createClient();
+
+        // Create Drizzle instance
+        export const db = drizzle({
+          client: gelClient, schema: {
+            ...schema,
+            ...relations
+          },
+        });
+
+        // Helper types for use in our application
+        export type Book = typeof schema.book.$inferSelect;
+        export type NewBook = typeof schema.book.$inferInsert;
+
+        export type Note = typeof schema.note.$inferSelect;
+        export type NewNote = typeof schema.note.$inferInsert;
+
+
+Perform database operations with Drizzle
+========================================
+
+.. edb:split-section::
+
+    Drizzle provides a clean, type-safe API for database operations. Here are some examples of common operations:
+
+    **Selecting data:**
+
+    .. code-block:: typescript
+
+        // Get all books with their notes
+        const allBooks = await db.query.book.findMany({
+          with: {
+            notes: true,
+          },
+        });
+
+        // Get a specific book
+        const book = await db.query.book.findFirst({
+          where: eq(books.id, id),
+          with: { notes: true },
+        });
+
+
+.. edb:split-section::
+
+    **Inserting data:**
```

Comment:
Maybe an example of doing a bulk insert here? That's one place where Drizzle actually really shines.